cmd/root.go

```diff
@@ -26,14 +29,41 @@ Prints any of the files in the shape that are missing from the repo.`,
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
 	RunE: func(cmd *cobra.Command, args []string) error {
-		shapeFile, err := cmd.Flags().GetString("shape-file")
+		shapeFilename, err := cmd.Flags().GetString("shape-file")
 		if err != nil {
 			return err
 		}
-		shapeSpec, err := ioutil.ReadFile(shapeFile)
-		if err != nil {
-			return err
+		if shapeFilename == "" {
+			return fmt.Errorf("shape file must be specified")
+		}
+
+		var shapeSpec []byte
+
+		// Check if the filename is actually a URL
+
+		_, err = url.Parse(shapeFilename)
+		if strings.HasPrefix(strings.ToLower(shapeFilename), "http") && err == nil {
```

Comment:
Since you already parsed it, might as well use the `Scheme` field and not have to manually call `strings.ToLower`.

Plus, I think it's a good idea to check the error first since it doesn't really matter what the prefix is if the url is malformed.

```suggestion
		u, err = url.Parse(shapeFilename)
		if err == nil && strings.HasPrefix(u.Scheme, "http") {
```